src/server/object_services/object_server.js

```diff
@@ -399,6 +399,7 @@ async function complete_object_upload(req) {
     }
 
     set_updates.create_time = new Date();
+    set_updates.cache_valid_time = new Date();
```

Comment:
Why are we adding this property unconditionally?
I expect it to be added only for caching buckets:
```suggestion
    if (req.bucket.namespace.caching) {
        set_updates.cache_valid_time = new Date();
    }
```

Reply:
Ok. Will make it conditional.